pom.xml

```diff
@@ -148,11 +149,11 @@
         <antlr.version>4.3</antlr.version>
         <lux.woodstox.version>4.1.1</lux.woodstox.version>
         <commons-codec.version>1.11</commons-codec.version>
-        <spring.version>4.3.10.RELEASE</spring.version>
+        <spring.version>5.0.4.RELEASE</spring.version>
         <!-- pax.web and jetty versions are closely tied together and should be upgraded together-->
-        <jetty.version>9.3.24.v20180605</jetty.version>
-        <pax.web.version>6.0.11</pax.web.version>
-        <pax.cdi.version>0.12.0</pax.cdi.version>
+        <jetty.version>9.4.11.v20180605</jetty.version>
+        <pax.cdi.version>1.0.0</pax.cdi.version>
+        <pax.web.version>7.2.3</pax.web.version>
```

Comment:
❓ Wasn't pax.web and jetty updated relatively recently? Is it already capable of another upgrade?

Reply:
Yes. We tend to try to use the version of pax.web that karaf uses. Technically this is a newer version that avoids the vulnerabilities that we want to avoid, but it's at least within the same major version that Karaf uses.